### PR TITLE
[ioctl/newcmd] build account update command line into new ioctl

### LINKED ISSUE #3002: [ioctl] build `account update` command line into new ioctl
The new ioctl (./ioctl/newcmd) is a refactored version of the old ioctl(./ioctl/cmd). It includes two major changes:
- The [client interface](https://github.com/iotexproject/iotex-core/blob/master/ioctl/client.go) is passed when constructing the Cobra command.
- Unit tests are to be added for each command.

The newcmd code currently is a draft. The code of old ioctl can be used for reference.

### What would you like to be added
add `accountupdate.go` and `accountupdate_test.go` in `./ioctl/newcmd/account`. 
**Code for reference**:
https://github.com/iotexproject/iotex-core/blob/master/ioctl/cmd/account/accountupdate.go

### PR DESCRIPTION
**Pull request by mmyj**
close #3002

build account update command line into new ioctl.

### CODE REVIEW

**Comment by mmyj** [comment]
PTAL @Liuhaai

File: ioctl/newcmd/account/accountupdate.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+			}
+
+			if CryptoSm2 {
+				// find the pem file and update
+				filePath, err := findSm2PemFile(addr)
+				if err != nil {
+					return output.NewError(output.ReadFileError, fmt.Sprintf("crypto file of account #%s not found", addr), err)
+				}
+
+				output.PrintQuery(fmt.Sprintf("#%s: Enter current password\n", acc))
```
Comment (Liuhaai) [inline ioctl/newcmd/account/accountupdate.go:68]: L68-L89 has similar pattern. It could be put into a subfunc

**Review by Liuhaai — APPROVED** [review]

**Comment by Liuhaai** [comment]
> PTAL @Liuhaai

thx for contribution

File: ioctl/newcmd/account/accountupdate.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+				}
+			}
+			output.PrintResult(fmt.Sprintf("Account #%s has been updated.", acc))
+			return nil
+		},
+	}
+	return cmd
+}
+
+func readSecretAndUpdate(client ioctl.Client, acc string,
```
Comment (mmyj) [inline ioctl/newcmd/account/accountupdate.go:116]: @Liuhaai I put the common logic into this. The caller will pass down the password checker and password updater into this sub-function.

File: ioctl/newcmd/account/accountupdate.go (hunk 1 above)
Comment (mmyj) [reply ioctl/newcmd/account/accountupdate.go]: Thanks for the suggestion.  ^_^

**Comment by millken** [comment]
Maybe it's clearer to use `account changePassword` here

**Review by millken — APPROVED** [review]

**Comment by mmyj** [comment]
> Maybe it's clearer to use `account changePassword` here

Sorry for slow response, I think your rename suggestion is better, Should I follow up and create a new PR for it?

**Pull request merged**